src/Smidge/Controllers/CompositeFileCacheFilterAttribute.cs

```diff
@@ -28,7 +28,7 @@ public IFilterMetadata CreateInstance(IServiceProvider serviceProvider)
         internal static bool TryGetCachedCompositeFileResult(ISmidgeFileSystem fileSystem, string cacheBusterValue, string filesetKey, CompressionType type, string mime, out FileResult result, out DateTime lastWriteTime)
         {
             result = null;
-            lastWriteTime = DateTime.Now;
+            lastWriteTime = DateTime.MinValue;
```

Comment:
This is a functional change - is this fixing another issue?

Reply:
This was meant to be moved to the bottom, updated.